evap/results/templates/result_bar.html

```diff
@@ -21,7 +21,11 @@
 >
 
     <div class="grade-bg-result-bar-count text-center{% if question_result.counts is None %} grade-bg-disabled{% endif %}">
-        <span class="fas fa-user small"></span> {{ question_result.total_count|default:0 }}
+        {% if course.can_publish_rating_results %}
+            <span class="fas fa-user small"></span> {{ question_result.total_count|default:0 }}
+        {% else %}
+            &mdash;
```

Comment:
sorry about this, but maybe we could use `<span class="far fa-eye-slash"></span>` here after all.